examples/mongodb-atlas/README.md

```diff
@@ -0,0 +1,199 @@
+# MongoDB Atlas
+
+[MongoDB Atlas](https://www.mongodb.com/atlas) is an integrated suite of cloud
+database and data services to accelerate and simplify how you build with data.
+It deploys and scales a MongoDB cluster in the cloud. MongoDB Atlas can be
+primarily classified under "MongoDB Hosting".
+
+## Prerequisites
+
+* Kubernetes 1.20+
+* Kanister controller version 0.90.0 installed in your cluster
+* Kanctl CLI installed (https://docs.kanister.io/tooling.html#install-the-tools)
+* MongoDB Atlas account registered with organization, project and cluster
+
+## Integrating with Kanister
+
+In case, if you don't have `Kanister` installed already, use following commands
+to install.
+```bash
+$ helm repo add kanister https://charts.kanister.io
+$ helm install kanister --namespace kanister --create-namespace \
+    kanister/kanister-operator --set image.tag=0.90.0
+```
+
+### Build docker image for MongoDB Atlas
+
+To build docker image for MongoDB Atlas, execute following steps. Image build is
+needed as connection process to MongoDB Atlas is using `mongosh` command. Also,
+it installs `atlas` CLI which is used for performing backup and restore
+operations. These gets installed in the image along with some other required
+packages in blueprint.
+
+```bash
+$ cd ~/kanister/docker/mongodb-atlas
+$ docker build -t <registry>/<account_name>/mongodb-atlas:<tag_name> .
+$ docker push <registry>/<account_name>/mongodb-atlas:<tag_name>
+```
+
+### Create Blueprint
+
+Create Blueprint in the same namespace as the controller
+
+**NOTE:**
+
+Replace `<registry>`, `<account_name>` and `<tag_name>` for the image with the
+values provided while building MongoDB Atlas docker image in
+./mongodb-atlas-blueprint.yaml before running following command.
+
+```bash
+$ kubectl create -f ./mongodb-atlas-blueprint.yaml -n kanister
+```
+
+### Create Secret
+
+Create a Secret which contains Atlas account details.
+
+```bash
+$ kubectl create namespace mongodb-atlas-test
+namespace/mongodb-atlas-test created
+
+$ kubectl create secret generic mongoatlassecret \
+    --from-literal=Public_Key="<public key from Atlas account>" \
+    --from-literal=Private_Key="<private key from Atlas account>" \
+    --from-literal=Org_Id="<organization ID from Atlas account>" \
+    --from-literal=Project_Id="<project ID from Atlas account>" \
+    --from-literal=Cluster_Name="<cluster name from Atlas account>" \
+    -n mongodb-atlas-test
+secret/mongoatlassecret created
```

Comment:
we should also mention are someone, who is following this guide, can figure these values for `Public_key` etc out.

Reply:
added details about how user can find these fields